project.clj

```diff
@@ -28,6 +28,8 @@
                  [ring/ring-core "1.7.0"]
                  [ring/ring-defaults "0.3.2"]
                  [ring-cors "0.1.12"]
+                 ;; APIKEY middleware
+                 [threatgrid/ring-api-key-middleware "0.1.2"]
```

Comment:
neat! lets keep an eye on the development of this middleware

Reply:
another good catch, I decided to not use that middleware after all, and just a simple wrap fn. Forgot to remove dependency, will do